test/integration/metrics/metrics_test.go

```diff
@@ -92,8 +114,8 @@ func TestApiserverMetrics(t *testing.T) {
 
 	// Make a request to the apiserver to ensure there's at least one data point
 	// for the metrics we're expecting -- otherwise, they won't be exported.
-	client := clientset.NewForConfigOrDie(&restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}}})
-	if _, err := client.CoreV1().Pods(metav1.NamespaceDefault).List(context.TODO(), metav1.ListOptions{}); err != nil {
+	client := clientset.NewForConfigOrDie(restclient.NewAggregatedConfig(&restclient.KubeConfig{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: &schema.GroupVersion{Group: "", Version: "v1"}}}))
```

Comment:
Why this is related to etcd upgrade?

Reply:
The code did not compile there since the aggregate rest config was introducced.

Reply:
But those tests are not used in our current test suites, right? Otherwise, it won't pass previous CI.

Reply:
I did run locally to  trigger the test cases. 

Not sure if we need to fix it now.